app/views/partials/_commentCard.html.erb

```diff
@@ -1,9 +1,13 @@
 <div class="horizontal-panel space-y flex flex-col border-black">
   <div class="flex flex-col px-5">
-   <% (1..items).each_with_index do |index| %>
-    <p class="text-lg">
-      Username: Comment <%= index %>
-    </p>
+    <% if comments.any? %>
+      <% comments.each do |comment| %>
+        <p class="text-lg">
+          Username: Comment: <%= comment.text %>
+        </p>
+      <% end %>
+    <% else %>
```

Comment:
- [ ] After adding a comment there is this username that comes in front of the comment did you mean a username of someone who posted a comment? if yes maybe you can add a field of the username to the form it will make more sense
![us](https://user-images.githubusercontent.com/44745855/150365514-68431f49-0625-4b1d-9d1f-88b2e9f84e49.png)